### PR TITLE
Bug fixes and validations for create Pool and RBD

### PR DESCRIPTION
**Pull request by cloudbehl**
- changed name property to "integration_name" for cluster
 - Fixed validation for creating a RBD
 - CSS changes for the create pool

### CODE REVIEW

**Comment by cloudbehl** [comment]
@kamleshverma1 @gnehapk please review.

File: src/commons/components/create-pool/create-pool.js [hunk 1]
```diff
@@ -87,22 +87,31 @@
...
+            vm.selectedCluster = vm.cephClusterList[0];
             vm.selectedType = vm.poolType[0];
             vm.selectedEC = vm.ECType[0];
             vm.selectedOwner = vm.owners[0];
             vm.updatePoolName();
         }
 
+        function getCephClusterList(){
+            var index;
+            for(index = 0 ; index < $rootScope.clusterData.clusters.length ; index++) {
```
Comment (gnehapk) [inline src/commons/components/create-pool/create-pool.js]: @cloudbehl Please store the length in some variable and then use it in for loop. This impacts the performance.

File: src/commons/components/create-pool/create-pool.js [hunk 2]
```diff
@@ -87,22 +87,31 @@
...
+                if($rootScope.clusterData.clusters[index].sds_name === 'ceph') {
```
Comment (gnehapk) [inline src/commons/components/create-pool/create-pool.js]: @cloudbehl Why do we need this condition.

File: src/commons/components/create-pool/create-pool.scss [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
+.create-pool-wizard {
+    .wizard-pf {
+			.wizard-pf-sidebar{
+				position: fixed;
+				left: 200px;
+				top: 200px;
+				bottom: 0;
+    	}
+    }
+
```
Comment (gnehapk) [inline src/commons/components/create-pool/create-pool.scss]: @cloudbehl please indent it properly.

File: src/modules/rbd/create-rbd/create-rbd.js [hunk 4]
```diff
@@ -118,32 +122,37 @@
...
-            vm.selectedCluster = vm.clusterList[0];
+            vm.selectedCluster = vm.cephClusterList[0];
             vm.selectedUnit = vm.sizeUnits[0];
             vm.updateRBDName();
             
         }
 
+        function getCephClusterList(){
+            var index;
+            for(index = 0 ; index < $rootScope.clusterData.clusters.length ; index++) {
```
Comment (gnehapk) [inline src/modules/rbd/create-rbd/create-rbd.js]: @cloudbehl Please store length in some variable.

File: src/commons/components/create-pool/create-pool.js (hunk 2 above)
Comment (cloudbehl) [inline src/commons/components/create-pool/create-pool.js]: yes we need it, because in drop down we only want to show the ceph clusters.

**Pull request merged**